docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,251 @@ sidebar_label: Wikidata
+
+Types are organized into an ontology, by specifying which types are subclasses of the others, using the [subclass of (P279)](https://www.wikidata.org/wiki/Property:P279) property. For instance, [public university (Q875538)](https://www.wikidata.org/wiki/Q875538) is a subclass of [university (Q3918)](https://www.wikidata.org/wiki/Q3918).
+
+One useful tool to visualize the subclasses of a given type is the [Wikidata Graph Builder](https://angryloki.github.io/wikidata-graph-builder/). For instance, here are [all the subclasses of academic institution (Q4671277)](https://angryloki.github.io/wikidata-graph-builder/?property=P279&item=Q4671277&mode=reverse).
+
+The reconciliation window allows you to select a type for your rows. This will restrict the matches to items that are instances of any subclass of the given type. For instance, if you select [public institution (Q875538)](https://www.wikidata.org/wiki/Q875538), then [university of Ljubljana (Q1377)](https://www.wikidata.org/wiki/Q1377) will be a possible match, because of the path above.
+
+By default, OpenRefine will propose you some types based on the first few items of your dataset. It is quite likely that these types will be too specific for your needs. You can specify a broader type that will encompass more items. For instance, if you have a dataset of universities, it might be better to pick [educational institution (Q2385804)](https://www.wikidata.org/wiki/Q2385804) rather than [university (Q3918)](https://www.wikidata.org/wiki/Q3918), for instance if some entry happens to be marked as a [college (Q189004)](https://www.wikidata.org/wiki/Q189004) in Wikidata.
+
+Some items are not marked as the instances of anything, because no one has taken the time to add these statements yet. If you restrict the reconciliation to a type, these items will not appear in the results, except when no items of the given type could be found for the particular name. In this case, items with no type can be returned: they will have a low score and will not be matched automatically.
+
+### Reconciling via unique identifiers
+
+You can supply a column of unique identifiers directly to Wikidata in order to pull more data, but these strings will not be "reconciled" against the external dataset. Matching identifiers does not validate them. 
+
+Apply the operation “Reconcile” → “Use values as identifiers” on your column of identifiers. All cells will appear as dark blue "confirmed" matches. You should check before this operation that the identifiers in the column exist on the target service.
+
+You may get false positives, which you will need to hover over or click on to identify.
```

Comment:
I know we've discussed this already but I think this is lacking a bit of context - I'd suggest that we explain what might happen if you use invalid identifiers rather than saying "false positives" which not everyone will understand

Reply:
Instead of explicating this too much here, I think we can just link to the equivalent section on the Reconciliation page? 

> You can supply a column of unique identifiers directly to Wikidata in order to pull more data, but [these strings will not be "reconciled" against the external dataset](reconciling#reconciling-with-unique-identifiers). These identifiers will not be validated. 
>
>Apply the operation “Reconcile” → “Use values as identifiers” on your column of identifiers. All cells will appear as dark blue "confirmed" matches. Some of the "matches" may be errors, which you will need to hover over or click on to identify.